plutus-metatheory/src/Utils.lagda.md

```diff
@@ -201,6 +224,9 @@ infixr 5 _∷_
 
 {-# COMPILE GHC List = data [] ([] | (:)) #-}
 
+```
+## DATA
+```
```

Comment:
"kinds" near the bottom of the file  looks as if it should be a header.

Reply:
Good spot :)